src/test/scala/com/crealytics/spark/excel/IntegrationSuite.scala

```diff
@@ -195,6 +196,48 @@ class IntegrationSuite extends FunSpec with PropertyChecks with DataFrameSuiteBa
         }
       }
 
+      it("writes data with legal disclaimer") {
+        val fileName = File.createTempFile("spark_excel_test_", ".xlsx").getAbsolutePath
+
+        val df =
+          Seq((1, "1", "2", "3"), (2, "4", "5", "6"), (3, "7", "8", "9")).toDF("id", "column1", "column2", "column3")
+
+        df.write
```

Comment:
Can you try to reuse more of the test infrastructure from the other tests? In order to do so, you could pass an additional argument Map which will get passed to the `option` or `options` method.
The test right now e.g. doesn't test the streaming version (`maxRowsInMemory = Some(20)`) which is working quite differently under the hood.